README.md

```diff
@@ -15,3 +15,11 @@ When adding images, please watch their size and weight. Compress them using some
 ## Adding a job offer
 
 We write job offers as we do with blog posts. Simply add the offer as a Markdown in [_jobs](https://github.com/coopdevs/coopdevs.github.io/tree/master/_jobs).
+
+## Development
+
+We use two branches: `master` holds only static files and serves the site through Github pages, while `develop` is the default branch. This one should be the target of all pull requests.
+
+## Deployment
+
+Once your pull request is approved, merge it. Then, from your local environment change the branch to `develop`, pull the changes and run `./deploy`. This will build and deploy `develop`'s contents to `master` making them live straight away.
```

Comment:
I think it's good to stress (again) the fact that the main branch is `develop`. 

```suggestion
Once your pull request is approved, merge it to `develop`. Then, from your local environment, change the branch to `develop`, pull the changes and run `./deploy`. This will build and deploy `develop`'s contents to `master` making them live straight away.
```

Reply:
Sure. I also thought it'd be good to refer to git flow. This is the name of the actual branching model, isn't it?

Reply:
git flow or github flow… not sure what to call it!
Although I’d say it’s a standard already in the industry, no?

Reply:
It is named git flow afaik http://danielkummer.github.io/git-flow-cheatsheet/